### PR TITLE
Included limited option to improve the planner

### PR DESCRIPTION
**Pull request by beatrizleon**

### CODE REVIEW

File: sr_multi_moveit/ur10srh_moveit_config/launch/moveit.rviz [hunk 1]
```diff
@@ -430,7 +430,7 @@ Visualization Manager:
         Show Robot Visual: true
         Show Trail: false
         State Display Time: 0.05 s
-        Trajectory Topic: move_group/display_planned_path
+        Trajectory Topic: /move_group/display_planned_path
```
Comment (toliver) [inline sr_multi_moveit/ur10srh_moveit_config/launch/moveit.rviz]: Why is it necessary to make the name absolute? Generally this is not recommended.

Comment (beatrizleon) [inline sr_multi_moveit/ur10srh_moveit_config/launch/moveit.rviz]: Ignore me. I thought it was needed.

File: sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml [hunk 2]
```diff
@@ -56,13 +56,13 @@ right_arm:
   planner_configs:
```
Comment (ugocupcic) [inline sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml:56]: no changes in that file right? It's better to remove that noise from the pr.

Comment (beatrizleon) [inline sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml:56]: I commented the planner "TRRTkConfigDefault" which is giving errors

Comment (ugocupcic) [inline sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml:56]: OK makes sense, but remove all the noise (other changes)

File: sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml [hunk 3]
```diff
@@ -56,13 +56,13 @@ right_arm:
...
     - SBLkConfigDefault
     - ESTkConfigDefault
-   # - LBKPIECEkConfigDefault
-   # - BKPIECEkConfigDefault
+   #- LBKPIECEkConfigDefault
```
Comment (ugocupcic) [inline sr_multi_moveit/ur10srh_moveit_config/config/ompl_planning.yaml]: like this noise

**Comment by ugocupcic** [comment]
@toliver I'm happy with that. Merge if you're happy too?

**Pull request merged**